03-members/dbhagen.yaml

```diff
@@ -0,0 +1,5 @@
+username: dbhagen
+push:
```

Comment:
@dbhagen you can only have one level of permissions for a certain repo, so keep `maintain` for `pulumi-fortios`.